python/ray/_raylet.pyx

```diff
@@ -186,8 +186,10 @@ cdef RayObjectsToDataMetadataPairs(
 cdef VectorToObjectRefs(const c_vector[CObjectReference] &object_refs):
     result = []
     for i in range(object_refs.size()):
-        result.append(ObjectRef(object_refs[i].object_id(),
-                                object_refs[i].call_site()))
+        result.append(ObjectRef(
+            object_refs[i].object_id(),
+            object_refs[i].owner_address().SerializeAsString(),
```

Comment:
Does this add extra overheads?

Reply:
This call takes about 1-2us, total `.remote()` time is ~200us. I think this is okay, but I can run microbenchmarks if necessary.